### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by ssgtcookie**
Signed-off-by: Niels <ssgtcookie@users.noreply.github.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 # Dependencies
 node_modules
+package-lock.json
```
Comment (Mcat12) [inline .gitignore]: We want to keep the package lock in the repo so we pin the versions.

**Comment by ssgtcookie** [comment]
Changes made.

**Comment by Mcat12** [comment]
The last commit should be made in a new branch since it's not directly tied to updating dependencies.

**Comment by ssgtcookie** [comment]
I dont have the rights to do that. I just followed your example by deving on the master branch like you did before, no bad intentions. :)

**Comment by dschaper** [comment]
Added `develop` branch, protected `master` and `develop`, changed PR base to `develop`.

**Comment by Mcat12** [comment]
Besides the last commit, I'm good with merging this.

**Comment by Mcat12** [comment]
Closing this PR for now. The dependency changes are likely out of date.

**Pull request closed without merging**